app/views/layouts/angular-bootstrap/_any_field.html.haml

```diff
@@ -0,0 +1,27 @@
+- prefix     ||= 'default'
+- input_type ||= 'text'
+- input_name ||= ''
+- input_desc ||= ''
+- input_emsg ||= ''
+- input_id   ||= [prefix, input_name].select(&:present?).join("_")
+
+- input_desc = _(input_desc) if input_desc.present?
+- input_emsg = _(input_emsg) if input_emsg.present?
```

Comment:
This is not going to work. These 2 lines above need to go away.

Reply:
Right, there are only a few very specific cases when `_()` is called on a non-constant.

In doubt, please refer to: http://manageiq.org/docs/guides/i18n

Reply:
ok